net/instaweb/rewriter/add_instrumentation.js

```diff
@@ -95,6 +95,12 @@ pagespeed.AddInstrumentation.prototype.sendBeacon = function() {
     if (timingApi['msFirstPaint']) {
       // IE.
       firstPaintTime = timingApi['msFirstPaint'];
+    } else if (window['performance']['timeOrigin'] &&
```

Comment:
Just for a bit of paranoia, could you make sure getEntriesByName is non-null before calling it?